### PR TITLE
Footer was modified

### PR DESCRIPTION
**Pull request by Tetiana-Dendemarchenko**

### CODE REVIEW

File: src/app/components/donate/donate.component.scss [hunk 1]
```diff
@@ -83,3 +108,12 @@
     @include box(50, 50);
   }
 }
+
+@media only screen and (max-width: 992px) {
+
+  .donate-button {
+    position: absolute;
+    top: 8px;
+    z-index: 100;
```
Comment (yana-tsylinska) [inline src/app/components/donate/donate.component.scss:117]: ??

File: src/app/components/donate/donate.component.ts [hunk 2]
```diff
@@ -18,4 +18,9 @@ export class DonateComponent implements OnInit {
       'Підтримка Черкаського академічного музично-драматичного театру імені Тараса Григоровича Шевченка',
       'http://theatre-shevchenko.ck.ua/assets/images/logo.png');
   }
+
+  openFile() {
+    const URL = 'https://d2m6zqv2ffku71.cloudfront.net/support-theatre.png';
+    window.open(URL, '_blank');
```
Comment (yana-tsylinska) [inline src/app/components/donate/donate.component.ts:24]: Make it with html

File: src/app/components/home-page/home-page.component.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <div class="home-page">
   <app-spinner name="home"></app-spinner>
 
-  <section class="poster-slider">
+<!--  <section class="poster-slider">
     <div class="poster-slider__component">
       <app-performance-slider></app-performance-slider>
     </div>
-  </section>
+  </section>-->
```
Comment (yana-tsylinska) [inline src/app/components/home-page/home-page.component.html:8]: Delete unused code

File: src/app/components/home-page/home-page.component.html [hunk 4]
```diff
@@ -38,7 +38,7 @@ <h1 class="we-offer__title" i18n="@@offerForYou">Offer for you</h1>
   </div>
   <app-actors-slider [randomEmployees]="randomEmployees"></app-actors-slider>
 
-  <section class="home-page-text__advantages">
+  <!--<section class="home-page-text__advantages">
```
Comment (yana-tsylinska) [inline src/app/components/home-page/home-page.component.html:41]: Why is it not deleted?

File: src/app/modules/core/components/footer/footer.component.html [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
+           aria-label="Facebook page"
+           title="Facebook page"
+           i18n-aria-label="@@facebookPage"
+           href="https://www.facebook.com/pg/%D0%A7%D0%B5%D1%80%D0%BA%D0%B0%D1%81%D1%8C%D0%BA%D0%B8%D0%B9-%D0%B0%D0%BA%D0%B0%D0%B4%D0%B5%D0%BC%D1%96%D1%87%D0%BD%D0%B8%D0%B9-%D0%BC%D1%83%D0%B7%D0%B8%D1%87%D0%BD%D0%BE-%D0%B4%D1%80%D0%B0%D0%BC%D0%B0%D1%82%D0%B8%D1%87%D0%BD%D0%B8%D0%B9-%D1%82%D0%B5%D0%B0%D1%82%D1%80-%D1%96%D0%BC%D0%B5%D0%BD%D1%96-%D0%A2%D0%B0%D1%80%D0%B0%D1%81%D0%B0-%D0%A8%D0%B5%D0%B2%D1%87%D0%B5%D0%BD%D0%BA%D0%B0-1672281213013382">
+          <img src="assets/images/facebook-box-fill.svg" class="facebook-box-fill" alt="Facebook logo">
+        </a>
       </li>
 
-      <li><span class="ucf-text" i18n="@@createdWithHelp">Site has been created with the help of</span>
+      <li class="ufc"><span class="ucf-text" i18n="@@createdWithHelp">Site has been created with the help of</span>
```
Comment (yana-tsylinska) [inline src/app/modules/core/components/footer/footer.component.html:23]: ufc? ucf? cuf? fuc?..

File: src/app/modules/core/components/footer/footer.component.html [hunk 6]
```diff
@@ -27,17 +32,12 @@
...
-        <a class="footer__link facebook__link"
-           target="_blank"
-           aria-label="Facebook page"
-           title="Facebook page"
-           i18n-aria-label="@@facebookPage"
-           href="https://www.facebook.com/pg/%D0%A7%D0%B5%D1%80%D0%BA%D0%B0%D1%81%D1%8C%D0%BA%D0%B8%D0%B9-%D0%B0%D0%BA%D0%B0%D0%B4%D0%B5%D0%BC%D1%96%D1%87%D0%BD%D0%B8%D0%B9-%D0%BC%D1%83%D0%B7%D0%B8%D1%87%D0%BD%D0%BE-%D0%B4%D1%80%D0%B0%D0%BC%D0%B0%D1%82%D0%B8%D1%87%D0%BD%D0%B8%D0%B9-%D1%82%D0%B5%D0%B0%D1%82%D1%80-%D1%96%D0%BC%D0%B5%D0%BD%D1%96-%D0%A2%D0%B0%D1%80%D0%B0%D1%81%D0%B0-%D0%A8%D0%B5%D0%B2%D1%87%D0%B5%D0%BD%D0%BA%D0%B0-1672281213013382">
-          <img src="assets/images/facebook-box-fill.svg" class="facebook-box-fill" alt="Facebook logo">
-        </a>
+      <li class="copy-right">
+        <div class="copy">
```
Comment (yana-tsylinska) [inline src/app/modules/core/components/footer/footer.component.html:36]: Why is it copy class name?

File: src/app/modules/core/components/header/header.component.scss [hunk 7]
```diff
@@ -1,133 +1,190 @@
...
     background-color: rgba($background-on-hover, .2);
   }
-}
 
-:host ::ng-deep {
+  @media only screen and (max-width: 1270px) {
 
-  .header-nav {
+    .search-icon-button {
+      right: 306px;
```
Comment (yana-tsylinska) [inline src/app/modules/core/components/header/header.component.scss:179]: Why 306?

File: src/app/modules/core/components/visually-impaired/visually-impaired.component.html [hunk 8]
```diff
@@ -43,7 +44,8 @@
           </div>
         </div>
       </div>
-      <img src="assets/images/close@3x.png" class="close-button" alt="Closing button" title="Close" i18n-title="@@ngb.alert.close"
+      <img src="assets/images/close 1.svg" class="close-button" alt="Closing button" title="Close"
```
Comment (yana-tsylinska) [inline src/app/modules/core/components/visually-impaired/visually-impaired.component.html:47]: Same to first error

**Review by yana-tsylinska — CHANGES_REQUESTED** [review]

**Comment by Tetiana-Dendemarchenko** [comment]
Broke this huge commit into smaller pieces. All the comments (by the team lead) about fixing the code were taken into consideration.

**Pull request closed without merging**